sale_partner_delivery_window/models/sale_order_line.py

```diff
@@ -80,8 +80,3 @@ def _prepare_procurement_values_time_windows(self, date_planned):
                 self.name,
             )
         return
-
```

Comment:
@grindtildeath do you remember why this was done?
Also, do you have a specific case where we want to keep the the UserError on `next_delivery_window_start_datetime`?
Can't we do anything better?